connect/src/com/telenor/connect/utils/ConnectUtils.java

```diff
@@ -49,4 +53,11 @@ public static void sendTokenStateChanged(boolean state) {
         intent.putExtra(LOGIN_STATE, state);
         localBroadcastManager.sendBroadcast(intent);
     }
+
+    private static Map<String, String> getErrorParams(String error, String errorDescription) {
```

Comment:
My github diff got a bit cluttered. You are actually calling getErrorParams twice. So maybe it make sense to leave it as it was the in the first commit. I'm sorry about that.